charts/super-agent-deployment/README.md

```diff
@@ -2,7 +2,7 @@
 
 # super-agent-deployment
 
-![Version: 0.0.16-beta](https://img.shields.io/badge/Version-0.0.16--beta-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.13.1](https://img.shields.io/badge/AppVersion-0.13.1-informational?style=flat-square)
+![Version: 0.0.17-beta](https://img.shields.io/badge/Version-0.0.17--beta-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.14.0](https://img.shields.io/badge/AppVersion-0.14.0-informational?style=flat-square)
```

Comment:
can we remove from the template the snippet generating this?

Reply:
I generated it with helm docs 🤔

Reply:
for example I've removed it from the other chart [here](https://github.com/newrelic/helm-charts/pull/1379/files#r1642502901). It was a @kang-makes idea to remove it so that we do not need to update it since it adds no real value